src/backend/marsha/bbb/utils/tokens.py

```diff
@@ -7,14 +7,20 @@
 from marsha.core.simple_jwt.tokens import ResourceAccessToken
 
 
-def create_classroom_stable_invite_jwt(classroom):
+def create_classroom_stable_invite_jwt(classroom, role="none", permissions=None):
```

Comment:
Is it possible to use the `NONE` constant here ? Available in the account model module